README.md

```diff
@@ -20,17 +20,18 @@
 ## Third Party Dependencies
 
 - snappy
-- lz4
 - zlib
 - thrift 0.7+ [install instructions](https://thrift.apache.org/docs/install/)
 - googletest 1.7.0 (cannot be installed with package managers)
 - Google Benchmark (only required if building benchmarks)
 
 You can install these dependencies using a package manager or using the
-`thirdparty/` scripts in this repository. On Homebrew, you can run:
+`thirdparty/` scripts in this repository.
+
+macOS already has zlib; on Homebrew, you can run:
```

Comment:
Fine as is, with the split sections per below I think it will be easy enough to follow